README.md

```diff
@@ -3,37 +3,23 @@
 OME-QtWidgets is a simple set of widgets and a viewer for simple image
 rendering using OME Files C++.
 
+Links
+-----
+
+- [Documentation] (http://www.openmicroscopy.org/site/support/ome-files-cpp/)
 
 Purpose
 -------
 
 Simple image rendering with OpenGL and Qt5 using OME Files C++ to read
 image pixel data and metadata.
 
-Supported formats
------------------
-
-OME-TIFF using the OME metadata model.
-
-For users
----------
-
-[Many software
-packages](http://www.openmicroscopy.org/site/support/bio-formats/users/index.html)
-use OME-Files to read and write microscopy formats.
-
-
-For developers
---------------
-
-You can use OME-Files C++ to easily support OME-TIFF in your software.
-
-
 More information
 ----------------
 
-For more information, see the [OME-Files web
-site](http://www.openmicroscopy.org/site/products/bio-formats).
+
+For more information, see the [documentation]
```

Comment:
Looks like an additional new line here, though it doesn't appear to affect the display